apps/prairielearn/src/components/CommentPopover.html.ts

```
@@ -0,0 +1,30 @@
+export function CommentPopover(comment: string | string[] | Record<string, any> | null) {
+  let content = '';
+  if (
+    !comment ||
+    (typeof comment === 'string' && comment.trim() === '') ||
+    (Array.isArray(comment) && comment.toString() === '[]') ||
+    (typeof comment === 'object' && Object.keys(comment).length === 0)
+  ) {
+    return '';
+  }
+  
+  const content = typeof comment === 'string' ? comment : JSON.stringify(comment, null, 2);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'content'. Consider to delete it or rename it.

'content' is defined here:


(lint/suspicious/noRedeclare)